eth-supply/src/lib.rs

```diff
@@ -0,0 +1,103 @@
+#[rustfmt::skip]
```

Comment:
For the modules could you use a modules folder layout instead? Have a look at some of the other substreams in our repo to see what I mean here

Reply:
Done!